allergies/allergies.html

```diff
@@ -0,0 +1,62 @@
+      <div class="col m8 s12">
+        <div class="Iconcontainer">
+          <div class="vl hide-on-small-only"> </div>
+          <div class="row">
+            <div value="peanut-free" class="col l3 m4 s4 notClicked healthIcon"><center><img id="peanut-freeIcon" class="responsive-img" src="../assets/white icons/peanut-free.png" style="width:58px; height:58px;"><p><strong>Peanut Free</strong></p></center></div>
+            <div value="sugar-conscious" class="col l3 m4 s4 notClicked healthIcon"><center><img id="sugar-consciousIcon" class="responsive-img" src="../assets/white icons/sugar-conscious.png" style="width:58px; height:58px;"><p><strong>Low sugar</strong></p></center></div>
+            <div value="vegetarian" class="col l3 m4 s4 notClicked healthIcon"><center><img id="vegetarianIcon" class="responsive-img" src="../assets/white icons/vegetarian.png" style="width:58px; height:58px;"><p><strong>Vegetarian</strong></p></center></div>
+            <div value="vegan" class="col l3 m4 s4 notClicked healthIcon"><center><img id="veganIcon" class="responsive-img" src="../assets/white icons/vegan.png" style="width:58px; height:58px;"><p><strong>Vegan</strong></p></center></div>
+            <div value="low-carb" class="col l3 m4 s4 notClicked dietIcon"><center><img id="low-carbIcon" class="responsive-img" src="../assets/white icons/low-carb.png" style="width:58px; height:58px;"><p><strong>Low Carb</strong></p></center></div>
+            <div value="low-fat" class="col l3 m4 s4 notClicked dietIcon"><center><img id="low-fatIcon" class="responsive-img" src="../assets/white icons/low-fat.png" style="width:58px; height:58px;"><p><strong>Low Fat</strong></p></center></div>
+            <div value="high-protein" class="col l3 m4 s4 notClicked dietIcon"><center><img id="high-proteinIcon" class="responsive-img" src="../assets/white icons/high-protein.png" style="width:58px; height:58px;"><p><strong>High Protein</strong></p></center></div>
+            <div value="tree-nut-free" class="col l3 m4 s4 notClicked healthIcon"><center><img id="tree-nut-freeIcon" class="responsive-img" src="../assets/white icons/tree-nut-free.png" style="width:58px; height:58px;"><p><strong>Tree Nuts</strong></p></center></div>
+            <div value="alcohol-free" class="col l3  offset-l3 m4 s4 notClicked healthIcon"><center><img id="alcohol-freeIcon" class="responsive-img" src="../assets/white icons/alcohol-free.png" style="width:58px; height:58px;"><p><strong>Alcohol-Free</strong></p></center></div>
+            <div value="balanced" class="col l3   m4 s12 notClicked dietIcon"><center><img id="balancedIcon" class="responsive-img" src="../assets/white icons/balanced.png" style="width:58px; height:58px;"><p><strong>Balanced</strong></p></center></div>
+          </div>
+
+          <!--this represents the buttons on the bottom of the screen and also shows this section's grid layout for 
+            different screen sizes-->
+          <div class="row">
+            <center>
```

Comment:
The <center> tag is considered obsolete. It's a best practice to use CSS to handle visual things like centering. Something like <div class="center"> would be better, with a .center css style to accomplish the centering.